src/plugins/english/freewebnovel.ts

```diff
@@ -115,7 +115,8 @@ class FreeWebNovel implements Plugin.PluginBase {
       loadedCheerio('div.txt').find('p:last-child').remove();
 
     const chapterText = loadedCheerio('div.txt').html() || '';
-    return chapterText;
+    return chapterText
+      .replace(/[ƒf][rɾг][ēeё][ēёe][wω][ёēe][bɓ][nɳη][ø૦ѳσo][ѵv][ёeē][lℓɭ]\.\s?[ƈcç][૦σøoѳ][๓ɱm]\.?/gi, '');
```

Comment:
Does this kind of string have a specific tag or classname?

Reply:
I am not sure. The website first loads the spam then removes it. Maybe to tackle web scrapers.

Reply:
> Does this kind of string have a specific tag or classname?

There is no tag/class name, it is baked directly into the <p> tag